### ISSUE TITLE
New Warning Produced by Android Runtime

### ISSUE THREAD

**Issue by ThinkDigitalRepair** [rxlabz/audioplayer#48]
W/MediaPlayer(29189): Use of stream types is deprecated for operations other than volume control
W/MediaPlayer(29189): See the documentation of setAudioStreamType() for what to use instead with android.media.AudioAttributes to qualify your playback use case

**Comment by eddywm** [comment]
I also noticed those warnings, you should probably propose a pull request since you seem to understand where it’s coming from.

Apparently you are also trying to build an audio player and you are relying on this library the same way I do.

What I'm doing is working on patches and fixes and try to propose PR here. Since the maintainers of this project are a little bit slow to merge you'll have to fork this repo and add those new features on your forked version till the maintainers come back.

**Comment by ThinkDigitalRepair** [comment]
I'm not actually sure where it's coming from, I just saw the errors pop up
:/

**Comment by ThinkDigitalRepair** [comment]
I fixed the errors and submitted a pull request. However, if I want to use these changes officially, would I have to create a forked pub as well?

**Comment by eddywm** [comment]
@ThinkDigitalRepair You'll have to fork the repository and refrence your version of this library in the yaml deps file waiting for the maintainers of this repo to merge your PR.

That's exactly what I did, waiting for my pull request #43 to get merged.

**Comment by ThinkDigitalRepair** [comment]
Yeah. But that takes it from pub.dartlang.org and not github, right? I
submitted a github pull request, but I can't use the updated code unless I
manually modify the package files in my editor, which is hack at best

**Comment by YDJSZY** [comment]
I have also this warning. I just run flutter in android emulators.

**Comment by softcoder** [comment]
Is steaming radio fixed for this control or should I find another package?
